src/gridfs.rs

```diff
@@ -51,7 +59,7 @@ pub struct FilesCollectionDocument {
     pub length: u64,
 
     /// The size of the file's chunks in bytes.
-    #[serde(rename = "chunkSize")]
+    #[serde(rename = "chunkSize", serialize_with = "u32_as_i32")]
```

Comment:
I believe you should be able to use [this method](https://docs.rs/bson/latest/bson/serde_helpers/fn.serialize_u32_as_i32.html) from `bson::serde_helpers` here

Reply:
Entirely forgot we had those, fixed :)